src/Tizen.NUI/src/public/Animation/AlphaFunction.cs

```diff
@@ -29,6 +29,8 @@ namespace Tizen.NUI
     public class AlphaFunction : Disposable
     {
 
+        internal global::System.Delegate keepDelegate;
```

Comment:
user가 정의한 delegate이기 때문에 userDelegate가 적당하지 않을까요?
object를 keep하기 위해서 추가한것이긴 하지만 keepDelegate는 해당 object를 잘 설명해주지 못하는것 같습니다.
```suggestion
        internal global::System.Delegate userDelegate;
```